sklearn/mixture/tests/test_gaussian_mixture.py

```diff
@@ -970,14 +970,14 @@ def test_sample():
 @ignore_warnings(category=ConvergenceWarning)
 def test_init():
     # We check that by increasing the n_init number we have a better solution
-    random_state = 0
-    rand_data = RandomData(np.random.RandomState(random_state), scale=1)
-    n_components = rand_data.n_components
-    X = rand_data.X['full']
+    for random_state in range(100):
```

Comment:
Could you double check that this runs quite quickly

Reply:
It runs in 4 seconds on my laptop. Is that sufficiently quick?  If not, we could reduce this to less than a second by iterating just 25 times. The probability for the unfixed code to pass this test would be 1/2^25, which is roughly 3e-8. Tell me what you prefer.

Reply:
Yes, 4s for this kind of test is excessively long, IMO. And if the unfixed code only has a chance of 3e-8 of passing (did you get that the right way around?), then we can run it for fewer than that. Alternatively, we can reduce the size of X if that's not actually related to the property being tested.

Reply:
Okay, reducing to 25 iterations. I think I got the calculation right: the unfixed code sets the `lower_bound_` to the lower bound of the last initialization, which has 50% chance of being higher than the lower bound of the first initialization. So in order to pass the updated test, the unfixed code would need to have 50% chance 25 times in a row, which 0.5^25 = 3e-8. In other words, the new test would catch the bug in the unfixed code.

Reply:
Sure, thanks. A smaller dataset would also be fine.